src/tools/emacs.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Emacs
+description: Use Dart on Emacs and its flavors
+toc: false
+---
+
+Emacs has a few major and minor modes built for Dart and Flutter development.
+
+All the steps are identical for enabling Flutter since
+[Dart-Snippets][dart-snippets] contain Flutter snippets too.
+[Flutter.el][flutter-mode] only allows you to run Flutter commands inside your Emacs.
+
+* [Dart-Mode][dart-mode] is the main mode for development.
+* [Dart-Snippets][dart-snippets] enables [Visual Studio Code](/tools/vs-code)
+  like snippets and more.
```

Comment:
As an Emacs user, I don't know what "Visual Studio Code like snippets" are. It would be good to elaborate.

Reply:
True.
I think a couple of examples will be helpful.
Will be added soon